setup.py

```diff
@@ -60,6 +65,44 @@ def get_mesa_templates(package, template_dir):
             )
 
 
+def get_frontend_dep():
+    leaflet_version = "1.8.0"
+    ensure_frontend_dep_single(
```

Comment:
If the output leaflet.js doesn't contain the version, the newer version won't be downloaded if you bump the version, because the file with the same name already exists.

Reply:
Yup that's true.